Documentation/deploy-master.md

```diff
@@ -179,24 +181,86 @@ spec:
     volumeMounts:
     - mountPath: /etc/ssl/certs
       name: ssl-certs-host
-	  readOnly: true
+      readOnly: true
   volumes:
   - hostPath:
       path: /usr/share/ca-certificates
     name: ssl-certs-host
 ```
 
+#### Set up the kube-podmaster
+
+The pod-master is responsible for implementing master-election for the kube-controller-manager and kube-scheduler. Because these services modify the cluster state, we only want to have one actor making modifications at a time.
```

Comment:
Can we call it `kube-podmaster` everywhere, rather than referring to it both as `kube-podmaster` and `the pod-master`?